include/swift/Parse/Token.h

```diff
@@ -266,6 +266,13 @@ class Token {
   bool isMultilineString() const {
     return MultilineString;
   }
+
+  bool isLifetimeDependenceToken() {
+    auto tokenRawText = getRawText();
+    return tokenRawText.equals("_copy") || tokenRawText.equals("_consume") ||
+           tokenRawText.equals("_borrow") || tokenRawText.equals("_mutate");
```

Comment:
I think it would be cleaner to check for `isContextualKeyword("_copy")` etc. It would be interesting to add a test case to check that eg. `` `_borrow`(x)``  errors.